maintainers/maintainer-list.nix

```diff
@@ -17331,6 +17331,11 @@
     githubId = 19433256;
     name = "Radoslaw Sniezek";
   };
+  rster2002 = {
+    name = "Bjørn";
+    github = "rster2002";
+    githubId = 26026518;
```

Comment:
Last question, are you sure you don't add an email address here? It will possibly become a standard in the future (https://github.com/NixOS/rfcs/pull/167), mainly to make it possible to contact with you when your account isn't available.